scripts-available/CDB_Conf.sql

```diff
@@ -0,0 +1,58 @@
+-- This will trigger NOTICE if CDB_CONF already exists
+DO LANGUAGE 'plpgsql' $$
+BEGIN
+    CREATE TABLE IF NOT EXISTS cartodb.CDB_CONF ( PARAM TEXT PRIMARY KEY, CONF TEXT NOT NULL );
```

Comment:
Detail for backups:
> Some extensions include configuration tables, which contain data that might be added or changed by the user after installation of the extension. Ordinarily, if a table is part of an extension, neither the table's definition nor its content will be dumped by pg_dump. But that behavior is undesirable for a configuration table; any data changes made by the user need to be included in dumps, or the extension will behave differently after a dump and reload. [...]

http://www.postgresql.org/docs/9.4/static/extend-extensions.html#AEN57458